.github/workflows/npm_publish

```diff
@@ -0,0 +1,32 @@
+# This workflow will run tests using node and then publish a package to npm when a release is created
+
+name: Publish npm Package
+
+on:
+  release:
+    types: [created]
```

Comment:
lazy question: I usually do a:
```
on:
  create:
    tags:
      - v*
```

What is the difference with `releases: types: [created]`?

Reply:
I really don't know 😜
This is just the template that GitHub provides for publishing an npm package  [Starter Workflow - NPM Publish](https://github.com/actions/starter-workflows/blob/fe49f6631698e7cd6ad2d94183791b61f9a59bfe/ci/npm-publish.yml)

A bit of research later: [Workflow Syntax - Events that trigger workflows](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#release)
➡️Functionally they seem to do the same (atleast in your context)
➡️Difference is, the one is triggered upon a **tag** starting with _v_. The other one is triggered upon creation of a **Release**

Looking further into it: [What's the difference between tag and release](https://stackoverflow.com/questions/18506508/whats-the-difference-between-tag-and-release)
➡️Release is a GitHub specific extension to a tag


Conclusively, I'd say, if you want to deeply integrate with GitHub use the one I suggested, If you want to stick with the git standard, use your suggestion 😄

Reply:
Thanks for doing my homework ;) To play a minor rebel, I think I prefer the `tag` one...

Reply:
I shall try on my fork to get it to work.
**Do not merge for now!**